packages/bridge/src/components/Withdraw/Withdraw.tsx

```diff
@@ -76,7 +76,7 @@ export const Withdraw: React.FC<WithdrawProps> = () => {
     return true;
   };
   const determineSwitchButton = () => {
-    if (depositNote && activeChain && activeChain.evmId != chainIdIntoEVMId(depositNote.note.targetChainId)) {
+    if (depositNote && activeChain && activeChain.evmId != internalChainIdIntoEVMId(depositNote.note.targetChainId)) {
```

Comment:
Let's future proof this for incoming substrate changes by adding another check on activeChain chainType:

```suggestion
    if (depositNote && activeChain && activeChain.chainType == ChainType.EVM && activeChain.evmId != internalChainIdIntoEVMId(depositNote.note.targetChainId)) {
```